code/game/atoms.dm

```diff
@@ -37,9 +37,6 @@
 	//Detective Work, used for the duplicate data points kept in the scanners
 	var/list/original_atom
 
-	//Detective Work, used for allowing a given atom to leave its fibers on stuff. Allowed by default
```

Comment:
Technically you can make items in hand also leave fibers, but this was never implemented. This has been here for 3+ years and never used anywhere else.